src-web/nls/platform.properties

```diff
@@ -146,30 +146,34 @@ modal.enforce-hcmcompliance.heading = Enforce policy
-modal.ansible.scheduleMode.info = Choose how often you want this automation to run. You can also choose to manually run it right now, which will save your change as is.
-modal.ansible.scheduleMode.manual.title = Manual run
-modal.ansible.scheduleMode.manual.info = Manual make this automation immediately run once. Once it runs, it will be set to disabled.
-modal.ansible.scheduleMode.once.title = Run once mode
-modal.ansible.scheduleMode.once.info = When this policy is violated, the automation will run one time. Once it runs, it will be set to disabled.
-modal.ansible.scheduleMode.disabled.title = Disable automation
-modal.ansible.scheduleMode.disabled.info = Automation won't run until you change this setting.
+
+ansible.automation.heading.create = Create policy violation automation
+ansible.automation.heading.edit = Edit policy violation automation
+ansible.automation.description.create = Create a new provider credentials to automate your selected policy and violation. You can also schedule the automation to run once, continuously, or disable for now.
+ansible.automation.description.edit = Modify an existing provider credentials to automate your selected policy and violation. You can also schedule the automation to run once, continuously, or disable for now.
+ansible.launch.connection = Launch to connection URL
+ansible.credential.selection.title = Provider credential
+ansible.credential.selection.placeholder = Select Ansible Credential
+ansible.no.credential = Ansible Credential is required to create Ansible Job, create your console here:
+ansible.launch.createCredential = Create a new provider credentials
+ansible.jobTemplates.selection.title = Job template
+ansible.jobTemplates.selection.placeholder = Select Ansible Job Template
+ansible.jobTemplates.editor.title = Ansible Job Template Extra Variables
+ansible.jobTemplates.editor.placeholder = Input Ansible Job Template Extra Variables Here
+ansible.no.jobTemplates.info = Failed to get Ansible Job Template information, please check related Ansible Tower and Ansible Job Template setting
+ansible.scheduleMode.title = Schedule automation
+ansible.scheduleMode.info = Choose how often you want this automation to run. You can also choose to manually run it right now, which will save your change as is.
+ansible.scheduleMode.manual.title = Manual run
+ansible.scheduleMode.manual.info = Manual make this automation immediately run once. Once it runs, it will be set to disabled.
+ansible.scheduleMode.once.title = Run once mode
+ansible.scheduleMode.once.info = When this policy is violated, the automation will run one time. Once it runs, it will be set to disabled.
+ansible.scheduleMode.disabled.title = Disable automation
+ansible.scheduleMode.disabled.info = Automation won't run until you change this setting.
+ansible.unsaved.data = You have made the change on configuration, do you want to save your policy automation? Or re-click 'Cancel' button to close.
```

Comment:
I suggest:

You made changes to the configuration. Do you want to save your policy automation? Click the Cancel button to close.

Instead of the single quotes, are you able to bold the word Cancel?